kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/clients/records/ConsumerRecord.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.systemtests.clients.records;
+
+import java.util.Map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class ConsumerRecord {
+    private static final Logger LOGGER = LoggerFactory.getLogger(ConsumerRecord.class);
+
+    protected String topic;
+    protected String key;
+    protected String payload;
+    protected int partition;
+    protected long offset;
+    protected Map<String, String> recordHeaders;
+
+    public String getTopic() {
+        return topic;
+    }
+
+    public String getKey() {
+        return key;
+    }
+
+    public String getPayload() {
```

Comment:
In Kafka parlance it is value.

Reply:
I'm going to change it, but I chose payload because it is what all clients return in the json file.